users/models.py

```diff
@@ -20,7 +24,7 @@ def __str__(self):
 class Listing(models.Model):
     title = models.CharField(max_length=100)
     description = models.CharField(max_length=100)
-    address = models.CharField(max_length=100)
+    address = models.CharField(max_length=100, blank=True)
```

Comment:
If we have decided to remove the address from the listing items, does it make sense to remove it completely from the model instead of hiding it?